src/automations/types/createAutomationQuery.ts

```diff
@@ -0,0 +1,37 @@
+import { AutomationAction } from '@/automations/types/actions'
+import { isRecord } from '@/utilities/object'
+
+export type CreateEventAutomationQuery = {
+  from: 'event',
+  event: { id: string, occurred: Date },
```

Comment:
One thing to note. In cloud this is type `Event` (a workspace event). But I don't want to move all the stuff associated with that type yet. So inlining just the two properties used here so this type is more narrow than `Event`.